Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Specify the version of Go to use
+FROM golang:1.19
+
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=linux \
```

Comment:
Why is it necessary to specify GOOS and GOARCH?

Reply:
But this would cause issues if you're running on arm64, right? In this case, you'd get a Docker container that's built on an arm64 base image but which has an app that runs on amd64.

Reply:
Changed to multi-platform step